ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,186 @@
+                                              checked="false"/>
+                                            <minOccur
+                                              jcr:primaryType="nt:unstructured"
+                                              wrapperClass="cmp-adaptiveform-panelcontainer__minOccur"
+                                              sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                              defaultValue="1"
+                                              fieldLabel="Minimum repetitions"
+                                              fieldDescription="Minimum number of times the vertical tabs appears on the form"
+                                              min="0"
+                                              name="./minOccur"/>
+                                            <maxOccur
+                                              jcr:primaryType="nt:unstructured"
+                                              wrapperClass="cmp-adaptiveform-panelcontainer__maxOccur"
+                                              sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                              emptyText="No Limit"
+                                              fieldLabel="Maximum repetitions"
+                                              fieldDescription="Maximum number of times the vertical tabs can appear on the form"
+                                              min="1"
+                                              name="./maxOccur"/>
+                                        </items>
+                                    </column>
+                                </items>
+                            </columns>
+                        </items>
+                    </repeat>
+                    <help
+                      jcr:primaryType="nt:unstructured"
+                      sling:resourceType="granite/ui/components/coral/foundation/include"
+                      path="core/fd/components/form/base/v1/base/cq:dialog/content/items/tabs/items/help"/>
+                    <accessibility
+                      jcr:primaryType="nt:unstructured"
+                      sling:resourceType="granite/ui/components/coral/foundation/include"
+                      path="core/fd/components/form/panelcontainer/v1/panelcontainer/cq:dialog/content/items/tabs/items/accessibility"/>
+                    <dor
+                      jcr:primaryType="nt:unstructured"
+                      sling:resourceType="granite/ui/components/coral/foundation/include"
+                      path="core/fd/components/form/panelcontainer/v1/panelcontainer/cq:dialog/content/items/tabs/items/dor"/>
+                    <properties
+                      jcr:primaryType="nt:unstructured"
+                      jcr:title="Properties"
+                      sling:orderBefore="cq:styles"
```

Comment:
cq:styles? this doesn't exist for core components, did we add in other components?

Reply:
yes it exists for other components too